src/main/java/com/automatedtest/sample/utils/Link.java

```diff
@@ -0,0 +1,14 @@
+package com.automatedtest.sample.utils;
+
+//TODO: support generic dynamic element
+public class Link {
```

Comment:
it doesn't have to be a Link every time, we should support every type of elements, you could name it DynamicControl or sth